.circleci/config.yml

```diff
@@ -1029,7 +1029,7 @@ workflows:
                 at: /
           workflow-name: "nightly-mainnet-migration-regression"
```

Comment:
nit: Will this test name also be changed?

Reply:
I will when we bring back the mainnet migration regression test.